File: app/src/main/java/com/v2/desafionubank/model/RequestReasonDetails.java
package com.v2.desafionubank.model;

/**
 * Created by CaioSChristino on 18/04/18.
 */

public class RequestReasonDetails {
    private String id;
    private boolean response;

    public RequestReasonDetails(ReasonDetails reasonDetails) {
        this.id = reasonDetails.getId();
        this.response = reasonDetails.isSelected();
    }
}

File: app/src/main/java/com/v2/desafio/model/RequestReasonDetails.kt
package com.v2.desafio.model

/**
 * Created by CaioSChristino on 18/04/18.
 */

class RequestReasonDetails(val reasonDetails: ReasonDetails) {
    private val id: String?
    private val response: Boolean

    init {
        this.id = reasonDetails.id
        this.response = reasonDetails.isSelected!!
    }
}